.github/ISSUE_TEMPLATE/server_error_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Server error
+about: Report a production bug from the server logs
+---
+<!--
+  - Before filing a new issue, please do a quick search to check that it hasn't
+  - already been filed on the [issue tracker](https://github.com/oppia/oppia/issues)._
+  -->
+
+This error occurred recently in production
+
+```
+Add error logs here.
+```
+
+**Additional context** Add details from the logs, like the page the error occurred on.
+
+**General instructions**
+There are no specific repro steps available for this bug report. The general procedure to fix server errors should be the following:
+
+* Analyze the code in the file where the error occurred and come up with a hypothesis of the reason.
+* Get the logic of the fix you propose validated (discuss on the issue thread).
+* Make the PR that fixes the issue, close the issue on merging the PR.
+* If the error reoccurs in production, the issue will be reopened for further investigation.
```

Comment:
Enclose this in parentheses and put this at the end of the previous bullet

Reply:
Done